content/F2015/F15-event-orientation.md

```diff
@@ -0,0 +1,25 @@
+title: Orientation Event and Dinner
+Date: 2015-09-13 15:00-18:00
+Category: Events
+Tags: social, big csters, academic
+Slug: orientation-F15
+Author: Fatema Boxwala
+Summary: Come out and learn about the university life, and get to know some
+awesome women in CS!
+
+Welcome to Waterloo, incoming women in CS! To help you get used to life here
+at Waterloo, WiCS is holding an orientation event, the last day before classes 
+start. This event is an opportunity for you to meet other women in your program
+as well as some awesome upper-years! There will be information sessions about
+various aspects about life here at Waterloo (ie. counselling, academic advisors,
+extracurriculars) as well as an opportunity to meet new people!
+
+Dinner will be provided! 
+
+## Event Details ##
+
++ **Who:** Incoming Women in CS
```

Comment:
"Incoming" applies to everyone returning to campus, probably better to be explicit.